common-items.lic

```diff
@@ -110,6 +109,7 @@ module DRCI
   ]
 
   @@put_away_item_failure_patterns = [
+    /^Stow what/
```

Comment:
Sorry, I approved too early 😅 
Need a comma now that this entry isn't the last one in the list
```suggestion
    /^Stow what/,
```

Reply:
Here's my quick sanity test with the "stow what" being moved -- all is good

```
> ,e echo DRCI.stow_item?('rope')

--- Lich: exec1 active.

[exec1]>stow my rope

Stow what?  Type 'STOW HELP' for details.
> 
[exec1: false]

--- Lich: exec1 has exited.

> get rope

You get some bundling rope from inside your armor pack.
> ,e echo DRCI.stow_item?('rope')

--- Lich: exec1 active.

[exec1]>stow my rope

You put your rope in your armor pack.
> 
[exec1: true]

--- Lich: exec1 has exited.

> get rope
drop rope

You get some bundling rope from inside your armor pack.
> 
You drop some bundling rope.

> ,e echo DRCI.stow_item?('rope')

--- Lich: exec1 active.

[exec1]>stow my rope

Stow what?  Type 'STOW HELP' for details.
> 
[exec1: false]

--- Lich: exec1 has exited.

> get rope

You pick up some bundling rope.

> lower ground

You lower the rope and place it on the ground at your feet.
> ,e echo DRCI.stow_item?('rope')

--- Lich: exec1 active.

[exec1]>stow my rope

You pick up the rope lying at your feet.
You put your rope in your armor pack.
> 
[exec1: true]

--- Lich: exec1 has exited.
```